target/linux/lantiq/xrx200/base-files/etc/board.d/02_network

```diff
@@ -130,7 +135,8 @@ lantiq_setup_macs()
 	avm,fritz7360sl)
 		wan_mac=$(macaddr_add "$(mtd_get_mac_binary urlader 0xa91)" 1)
 		;;
-	avm,fritz7362sl)
+	avm,fritz7362sl|\
+	avm,fritz3390)
```

Comment:
Please apply alphabetic sorting.

Reply:
I'm not sure how to sort this. Just inside the case, i.e. swap 3390 and 7362sl, or both before 7360sl?

Reply:
```
	avm,fritz3390|\
	avm,fritz7362sl)
		...
```
and then move the whole block up so it comes before fritz7360sl ...